src/main/java/org/opentripplanner/routing/core/ItineraryFares.java

```diff
@@ -0,0 +1,120 @@
+package org.opentripplanner.routing.core;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ImmutableMultimap;
+import com.google.common.collect.Multimap;
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.Currency;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import org.opentripplanner.ext.fares.model.FareProduct;
+import org.opentripplanner.ext.fares.model.LegProducts;
+import org.opentripplanner.model.plan.Leg;
+import org.opentripplanner.util.lang.ToStringBuilder;
+
+/**
+ * <p>
+ * Fare is a set of fares for different classes of users.
```

Comment:
Javadoc should reflect the change in the name

Reply:
Done.